src/Core/Graphics/Sprite.cpp

```diff
@@ -512,6 +512,30 @@ namespace obe::Graphics
             }
         }
 
+        Color spriteColor = Color();
```

Comment:
No need to do a
```cpp
Color spriteColor = Color();
```
```cpp
Color spriteColor;
```
will already use the default constructor.
(Also maybe rename it to `color` ?)